### ISSUE TITLE
Two ENIGMA variants translated incorrectly from HGVS

### ISSUE THREAD

**Issue by melissacline** [BD2KGenomics/brca-exchange#153]
In the second batch of ENIGMA submissions, there are variants translated as chr13:32339591:T>T and chr13:32340804:A>A respectively.  Respectively, they were specified as c.5237_5238dup and c.6450dup.  More correctly, these should've been 5237_5238dupT and c.6450dupA.  What was the source of this bug, is it a limitation of the HGVS translation libraries?

**Comment by lmallin** [comment]
The HGVS from the original files received from ENIGMA were c.5237_5238dup and c.6450dup.  After checking the updated ENIGMA files with the ClinVar accessions, another column 'preferred_variant_name' was included which stated the variants as c.5237_5238dupCT and c.6450dupA, respectively.  I checked these HGVS strings with the associated ClinVar records and they matched up.  Since it seems to be a typo or something in the original files,  I updated the HGVS to c.5237_5238dupCT and c.6450dupA.
